xarray/test/test_dask.py

```diff
@@ -300,3 +300,31 @@ def test_stack(self):
                              dims=['w', 'z'])
         assert stacked.data.chunks == expected.data.chunks
         self.assertLazyAndIdentical(expected, stacked)
+
+    def test_dot(self):
```

Comment:
This has the right idea, but it's a pretty complex example. I would prefer something more minimal like I suggested earlier, e.g., 
```
def test_dot(self):
    eager = self.eager_array.dot(self.eager_array[0])
    lazy = self.lazy_array.dot(self.lazy_array[0]))
    self.assertLazyAndAllClose(eager, lazy) 
```